R/run.R

```diff
@@ -51,13 +51,22 @@ run_tutorial <- function(name = NULL, package = NULL, shiny_args = NULL) {
     )
   }
 
+  # set rmarkdown args to render in tmp dir
+  temp_output_dir <- tempdir()
```

Comment:
Let's namespace the folders with the package and name inside a `"learnr"` folder.  (And create the folder if it doesn't exist.

```suggestion
  temp_output_dir <- file.path(tempdir(), "learnr", package, name)
  if (!dir.exists(temp_output_dir)) {
    dir.create(temp_output_dir, recursive = TRUE)
  }
```